File: qe-selenium/qe-selenium-java/src/main/java/com/demos/Activity1.java
package com.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {
        FirefoxDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net");
        System.out.println(driver.getTitle());
        System.out.println();
        driver.findElement(By.linkText("About Us")).click();
        System.out.println(driver.getTitle());

        driver.quit();
    }
}

File: qe-selenium/qe-selenium-python/Activity1.py
from selenium.webdriver.common.by import By
from selenium import webdriver

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net")
    
    driver.find_element(By.XPATH,'/html/body/div/main/div/div/main/a').click()
    print(driver.title)
    driver.quit

